README.md

```diff
@@ -1,135 +1,239 @@
 
-🚫This is just an example. Replace this with your data model
+```
+  input UpdateEventInput {
+    id: ID
+    Date: String
+    Start_Time: String
+    End_Time: String
+    Title: String
+    Description: String
+    Photo: String
+    category_id: Int
+    user_id: Int
+    Modifiers: String
+    Address: String
+    Latitude: Float
+    Longitude: Float
+  }
+```
 
-#### 2️⃣ ORGANIZATIONS
+#### Category Type and Inputs
 
 ---
 
 ```
-{
-  id: UUID
-  name: STRING
-  industry: STRING
-  paid: BOOLEAN
-  customer_id: STRING
-  subscription_id: STRING
-}
+  type Category {
+    id: ID!
+    Category: String!
+  }
 ```
 
-#### USERS
-
----
-
 ```
-{
-  id: UUID
-  organization_id: UUID foreign key in ORGANIZATIONS table
-  first_name: STRING
-  last_name: STRING
-  role: STRING [ 'owner', 'supervisor', 'employee' ]
-  email: STRING
-  phone: STRING
-  cal_visit: BOOLEAN
-  emp_visit: BOOLEAN
-  emailpref: BOOLEAN
-  phonepref: BOOLEAN
-}
+  input NewCategoryInput {
+    id: ID
+    Category: String!
+  }
 ```
 
 ## 2️⃣ Actions
 
-🚫 This is an example, replace this with the actions that pertain to your backend
-
-`getOrgs()` -> Returns all organizations
+`query getAllUsers` -> Returns all users
 
-`getOrg(orgId)` -> Returns a single organization by ID
+`query getUserById(id: ID!)` -> Returns a single user by ID
 
-`addOrg(org)` -> Returns the created org
+`mutation addUser(input: NewUserInput!)` -> Returns a created user
 
-`updateOrg(orgId)` -> Update an organization by ID
+`mutation updateUser(id: ID!, input: UpdateUserInput!)` -> Update a user by ID
 
-`deleteOrg(orgId)` -> Delete an organization by ID
+`mutation removeUser(id: ID!)` -> Delete a user by ID
 <br>
 <br>
 <br>
-`getUsers(orgId)` -> if no param all users
 
-`getUser(userId)` -> Returns a single user by user ID
+`query getAllEvents` -> Returns all events
 
-`addUser(user object)` --> Creates a new user and returns that user. Also creates 7 availabilities defaulted to hours of operation for their organization.
+`query getAuthoredEvents(id: ID!)` -> Returns all events from user by his/her user id
 
-`updateUser(userId, changes object)` -> Updates a single user by ID.
+`query getEventById(id: ID!)` -> Returns a single event by event id
 
-`deleteUser(userId)` -> deletes everything dependent on the user
+`mutation addEvent(input: NewEventInput!)` --> Creates a new event and returns that event
+
+`mutation updateEvent(id: ID!, input: UpdateEventInput!)` -> Updates a single event by ID
+
+`mutation deleteEvent(id: ID!)` -> deletes a sinlge event by id
+<br>
+<br>
+<br>
+
+`query getAllCategories` -> Returns all categories
+
+`query getCategoryById(id: ID!)` -> Returns a single category by category id
+
+`mutation addCategory(input: NewCategoryInput!)` -> Creates a new category and returns that category
 
 ## 3️⃣ Environment Variables
 
 In order for the app to function correctly, the user must set up their own environment variables.
 
 create a .env file that includes the following:
```

Comment:
It might be best to reference the `.env.example` file which should be maintained with examples of all parameters.

Reply:
Good call. I changed this section so that it references the .env.example file. This should help keep the readme cleaner.